docs-content/general/6_product-features/6_metrics/2_metrics-tutorials/3_user-engagement.md

```diff
@@ -0,0 +1,56 @@
+---
+toc: User Engagement
+title: Creating User Engagement Metrics
+slug: user-engagement
+---
+
+<EmbeddedVideo 
+  src="https://www.youtube.com/embed/7BhzaEVqsS4"
+  title="Metrics Tutorial: Measuring User Page Clicks"
+  allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+/>
+
+## Overview
+
+This tutorial guides you through creating a graph to measure and analyze user engagement metrics. By following these steps, you'll be able to track user interactions with your application and gain insights into user behavior and trends.
+
+## Step-by-step Guide
+
+### 1. Select the Data Source
+
+Begin by choosing the source of the metrics. For user engagement, we'll use user sessions as our data source. This will provide us with rich information about how users interact with our application.
```

Comment:
Idea: `Begin by choosing the "metrics" source.` Thought it may be a little confusing with the overall metrics product.